db/models/spots.js

```diff
@@ -3,6 +3,7 @@ import mongoose from "mongoose";
 const { Schema } = mongoose;
 
 const AdditionalInformationSchema = new Schema({
+  _id: { type: Schema.Types.ObjectId },
```

Comment:
I can see you're explicitly managing the `_id` key and manually saving/updating/deleting to MongoDB (in your endpoints), however MongoDB manages its `ids` internally. You don't need to do it yourself.

Is there a reason for doing everyting manually?

Reply:
Good question 😅 the reason is that i had serious problems getting it work yesterday. So i tried to define the key manually. now that it works, it works without the manual key. thanks for the hint